governance/polls/templates/Atlas/Atlas Edit Monthly Cycle Proposal - $date_MONTH_DD,_YYYY.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Atlas Edit Monthly Cycle Proposal - $date_MONTH_DD,_YYYY
```

Comment:
Monthly Cycle AEP are numbered. We probably should have the number as part of the title.

Reply:
https://github.com/makerdao/community/pull/2750/commits/f173b8e84185aeec30d18276c5b4d09de977896b